Bio/Crystal/__init__.py

```diff
@@ -3,11 +3,37 @@
 # license.  Please see the LICENSE file that should have been included
 # as part of this package.
 
-"""Represent the NDB Atlas structure (a minimal subset of PDB format).
+"""Represent the NDB Atlas structure (a minimal subset of PDB format) (OBSOLETE).
 
 Hetero, Crystal and Chain exist to represent the NDB Atlas structure.  Atlas
-is a minimal subset of the PDB format.  Heteo supports a 3 alphameric code.
+is a minimal subset of the PDB format.  Hetero supports a 3 alphanumeric code.
 The NDB web interface is located at http://ndbserver.rutgers.edu
+
+There is no use of the classes defined on Bio.Crystal on Biopython,
+so the module will be removed.
+
+Bio.Crystal.Hetero substitute is Bio.PDB.Atom
+Bio.Crystal.Chain substitute is Bio.PDB.Chain
+Bio.Crystal.Crystal substitute is Bio.PDB.Structure
+
+Using Bio.PDB you can navigate the data as below::
+
+    from Bio.PDB.PDBParser import PDBParser
+    parser = PDBParser(PERMISSIVE=1)
+    # PDB NDB Only file
+    structure = parser.get_structure("001", "001_msd.pbd")
+    for model in structure:
+        print('Model ',model)
+        for chain in model:
+            print('Chain ', chain)
+            for residue in chain:
+                print('Res ', residue)
+                for atom in residue:
+                    print('Atom ', atom)
+
+Since there is no use of the classes and their function are already
+covered by Bio.PDB, Bio.Crystal will be deprecated.
```

Comment:
Replace with: Bio.Crystal is self-contained, with the main functionality covered by Bio.PDB.